src/main/java/uk/gov/hmcts/reform/pip/data/management/service/PublicationService.java

```diff
@@ -68,10 +74,22 @@ public Artefact createPublication(Artefact artefact, MultipartFile file) {
             file
         );
         artefact.setPayload(blobUrl);
-
         return artefactRepository.save(artefact);
     }
 
+
+    /**
+     * Checks if the artefact has a display from date of today or previous then triggers the sub fulfilment
+     * process on subscription-management if appropriate.
+     */
+    public List<Subscription> checkAndTriggerSubscriptionManagement(Artefact artefact) {
+        if (artefact.getDisplayFrom().isBefore(LocalDateTime.now().plusDays(1))) {
+            return subscriptionManagementService.sendSubTrigger(artefact);
+        } else {
+            return Collections.emptyList();
```

Comment:
Don't believe we need the triggered subs in data management.

Reply:
sorted